packages/node_modules/@webex/widget-speed-dial/src/SpeedDials.tsx

```diff
@@ -59,9 +60,14 @@ const SortableItem = SortableElement<ISpeedDialProps>(
         if (e.key === 'Enter' && isContactFocused) {
           childRef.current?.handleEnterKey(e);
         } else if (e.shiftKey && e.key === 'F10' && isContactFocused) {
+          childRef.current?.setIsContextMenuVisible(true);
```

Comment:
Next time when you raise a PR, could you please create an ENUM for keyboard keys such as F10, Enter, Escape, and Tab?

Reply:
Sure Akula